securedrop/journalist_app/utils.py

```diff
@@ -319,14 +319,51 @@ def col_delete(cols_selected: List[str]) -> werkzeug.Response:
         db.session.commit()
 
         num = len(cols_selected)
-        flash(ngettext('{num} collection deleted', '{num} collections deleted',
-                       num).format(num=num),
-              "notification")
+        flash(
+           Markup(
+             ngettext(
+               '<b>Success!</b> The account and all data for {num} source have been deleted.',
+               '<b>Success!</b> The accounts and all data for {num} sources have been deleted',
+               num).format(num=num)),
+           "success")
+
+    return redirect(url_for('main.index'))
+
+
+def delete_source_files(filesystem_id: str) -> None:
+    """deletes submissions and replies for specified source"""
+    source = get_source(filesystem_id, include_deleted=True)
+    if source is not None:
+        # queue all files for deletion and remove them from the database
+        for f in source.collection:
+            try:
+                delete_file_object(f)
+            except Exception:
```

Comment:
Is it worth logging errors here? The delete-disconnected-submissions scripts will catch these. If you think it's useful, it might be worth documenting that it's why we are `pass`ing here

Reply:
I had a pass rather than handling (or at least logging it) because originally the source directory was also getting nuked. the disconnected submissions job *will* catch extra files, but if files are already missing then the FileNotFoundError here will not be caught iirc. since we're deleting anyway this isn't a problem to pass IMO